packages/superset-ui-chart/src/clients/ChartClient.ts

```diff
@@ -6,19 +6,17 @@ import {
   Json,
   SupersetClientClass,
 } from '@superset-ui/connection';
+import { AnnotationLayerMetadata, QueryFormData, Datasource } from '@superset-ui/query';
 import getChartBuildQueryRegistry from '../registries/ChartBuildQueryRegistrySingleton';
 import getChartMetadataRegistry from '../registries/ChartMetadataRegistrySingleton';
-import { AnnotationLayerMetadata } from '../types/Annotation';
```

Comment:
Judging from the name, `AnnotationLayerMetadata` seems more related to charts than queries. why is it being pulled out here?